homeassistant/components/sensor/envirophat.py

```diff
@@ -53,7 +53,13 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the Sense HAT sensor platform."""
-    data = EnvirophatData(config.get(CONF_USE_LEDS))
+    try:
```

Comment:
Yes, the home-assistant core will check the import error. So you need only include your library at this point.